sencha-workspace/packages/slate-cbl/src/widget/Popover.js

```diff
@@ -3,19 +3,30 @@ Ext.define('Slate.cbl.widget.Popover', {
     extend: 'Ext.Component',
     xtype: 'slate-cbl-popover',
 
+    config: {
+        pointer: 'left'
+    },
 
     autoEl: 'aside',
     componentCls: 'popover',
-    cls: 'point-left',
     defaultAlign: 'tl-tr',
     floating: true,
     tpl: [
-        '<h1 class="popover-title">',
-            '<svg class="popover-pointer-img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 5 10" aria-hidden="true" role="presentation">',
-                '<polygon class="popover-pointer" points="0,5 5,0 5,10" />',
-            '</svg>',
-            '{title}',
-        '</h1>',
+        '<tpl if="title">',
+            '<h1 class="popover-title">',
+                '<svg class="popover-pointer-img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 5 10" aria-hidden="true" role="presentation">',
+                    '<polygon class="popover-pointer" points="0,5 5,0 5,10" />',
+                '</svg>',
+                '{title}',
+            '</h1>',
+        '</tpl>',
         '<div class="popover-body">{body}</div>'
-    ]
+    ],
+
+    updatePointer: function(newPointer, oldPointer) {
+        var me = this;
+
+        me.removeCls('point-' + oldPointer);
```

Comment:
there is a toggleCls method, but oldPointer and newPointer here may be different so this seems correct. Although when pointer gets its initial value oldPointer will be null so this should be wrapped in if (oldPointer)